### PR TITLE
Blog post about NASA award

### PR DESCRIPTION
**Pull request by hayesla**
anything missing?

### CODE REVIEW

File: posts/2024/2024-06-19-groupaward.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+date: Jun 19 2024
+author: Laura Hayes
+category: Update
+---
+
+# SunPy Development Team wins a NASA Group Achievement Award!
+
+We are delighted to announce that the SunPy development team has been awarded with a NASA Group Achievement Award! 
+
+The award was given to our team for "developing and continually improving free and reliable open-source software that supports NASA missions and the analysis of NASA’s vast archives of solar physics data".
```
Comment (Cadair) [inline posts/2024/2024-06-19-groupaward.md:12]:
```suggestion
The award was given to our team for "developing and continually improving free and reliable open-source software that supports NASA missions and the analysis of NASA's vast archives of solar physics data".
```

??

**Review by Cadair — APPROVED** [review]

**Comment by nabobalis** [comment]
pre-commit.ci autofix

**Comment by hayesla** [comment]
ah the picture doesnt show https://sunpy.org/posts/2024/2024-06-19-groupaward/ 

did I do this wrong? 

```
<div style="text-align: center;">
    <img src="sunpy_award.jpg" width="500" />
</div>
``` ?

**Comment by nabobalis** [comment]
I am looking into and will push a fix when I can.

**Comment by nabobalis** [comment]
It is now fixed. I just switched out the html code for markdown image display and that did the trick.

Not sure why the html didnt work.

**Pull request merged**